modules/041-linstor/templates/linstor-affinity-controller/deployment.yaml

```diff
@@ -95,6 +95,13 @@ spec:
                 secretKeyRef:
                   name: linstor-client-https-cert
                   key: ca.crt
+          resources:
+            requests:
+              {{- include "helm_lib_module_ephemeral_storage_only_logs" . | nindent 14 }}
```

Comment:
```suggestion
{{- include "helm_lib_module_ephemeral_storage_only_logs" . | nindent 14 }}
```

Reply:
internally we did not come to a consensus about if/else constructions, this module is constructed with this style.
I suggest to keep it as is until we'll have common decision and refactor this module afterwards.